components/21-atoms/file/_file.scss

```diff
@@ -14,21 +13,25 @@
     @include button-hover;
 
     max-width: 100%;
-    margin: 0 .6rem;
-    padding-top: .6rem;
-    padding-bottom: .6rem;
+    margin: 0 .75rem 0 .6rem;
+    padding: .4rem .75rem;
     content: attr(data-text);
   }
 
   &::after {
+    @include theme('color', 'default-text-color--darken-1', 'file-selection');
```

Comment:
`file-selection` is te algemeen. Dit gaat over de "color", dat verwerk je best in de naam. Kijk naar reeds bestaande voorbeelden in de blue theme.

Reply:
Ik heb de class naam veranderd naar no-file-selected-text-color dit leek mij ook duidelijker als naam aangezien het specifiek over de niet toegevoegd tekst gaat.